b_cfn_elasticsearch_cloner/resource.py

```diff
@@ -30,6 +30,9 @@ def __init__(
         elasticsearch_index: ElasticsearchIndexResource,
         dynamodb_table: Table,
         kms_key: Optional[Key] = None,
+        embeddings_key: str = None,
+        sagemaker_endpoint_name: str = None,
+        sagemaker_embeddings_key: str = None
```

Comment:
It would be great if you would add a small docstring at least for these three new arguments.